### PR TITLE
BLAC-2414 Enforce checklist-files

### PR DESCRIPTION
**Pull request by alexandresavicki**
### :key: Purpose

This PR implements global workflows for entire organization using github actions. We are using an plugin [organization-workflows](https://github.com/SvanBoxel/organization-workflows) to do that. But we opted for deploy and maintain it on our [infrastructure](https://github.com/quintoandar/infrastructure/pull/3772), because it's needs an data source to persistence (source repository, check id, sha, and run id), so we decide build our own [docker-oragnization-workflow](https://github.com/quintoandar/docker-github-organization-workflow)

### :hammer:  Effects
 
 This initial workflow will be enforce the checklist file inside repositories that creates any resource on our K8s clusters, since one pull request are submited for an repository, an action will be trigged and check if folder `kube` exists than enforce PR owner fill the `.checklist.yaml` at root of your repositorie.
 Also we are forcing some values inside the checklist like:
`cost_center`, `department`, `ecosystem`, `owner`, `repo_status`, `service`, `squad`, `tribe`, `channels_squad`, `channels_alerts`

### :warning: Validation
Initially we have a toggle to disable script just set `EXIT CODE ERROR = 0` or` EXIT CODE ERROR = 1` to enable / disable it.
 
 See bellow some screenshots:
 *Example of PR*
PR Example

*Example of Error*
Error Example

*Example of Success*
Pass Example

 ### :bulb: Tip 
With this implementation we can run scripts for all repositories inside Quintoandar organization, without necessity include an Action on each repository one by one.

### CODE REVIEW

File: .github/workflows/enforce-checklist-file.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+name: Enforce checklist file
+
+on:
+  push:
+    branches: [ $default-branch ]
+  pull_request:
+    branches: [ $default-branch ]
+  repository_dispatch:
+    types: [org-workflow-bot]
```
Comment (thspinto) [inline .github/workflows/enforce-checklist-file.yml:9]: I think it's nice to have a comment explaining this type. I'll certainly forget why we need this in two weeks.

**Review by thspinto — COMMENTED** [review]
Should we add a yaml linter in this repo to make all workflows follow the same rules?

**Comment by alexandresavicki** [comment]
> Should we add a yaml linter in this repo to make all workflows follow the same rules?

We can add in another moment, i need figure out a pattern to lint our workflows.

**Review by thspinto — APPROVED** [review]

File: scripts/enforce-checklist-file.sh [hunk 2]
```diff
@@ -0,0 +1,204 @@
+#!/bin/bash
+
+#Toglle to disable Script set 1 to enable
```
Comment (gabriel-dantas98) [inline scripts/enforce-checklist-file.sh:3]: #
```suggestion
#Toggle to disable Script set 1 to enable
```

**Review by gabriel-dantas98 — APPROVED** [review]
Niceee!  🚀 🔝 

I think that in the future we can centralize this data in one place so that we can update all our data checks for ecosystems, departments and cost centers.

**Review by rkist — APPROVED** [review]

**Pull request merged**